model/OrganisationConfig.ts

```diff
@@ -22,3 +22,16 @@ export interface OrganisationConfig {
 	portalHideMultipleUseOption?: boolean
 	securityConfig?: SecurityConfig
 }
+
+export namespace OrganisationConfig {
+	export function is(value: OrganisationConfig | any): value is OrganisationConfig {
+		return (
+			value == undefined ||
+			(typeof value == "object" &&
+				(value.defaultModulrUsage == undefined ||
+					value.defaultModulrUsage == "SINGLE_USE" ||
+					value.defaultModulrUsage == "SINGLE_USE_ALLOW_TEST_AUTH" ||
+					value.defaultModulrUsage == "MULTIPLE_USE"))
```

Comment:
looks like this needs some more things moving forward, maybe this is fine with how it's about to be used for the time being

Reply:
also those values look like CardUsage (which maybe has an is-function)

Reply:
Changed to use CardUsage.
Missing props will be added later.